tests/transform_source/build.bp

```diff
@@ -91,9 +91,22 @@ bob_binary {
     srcs: ["main.cpp"],
 }
 
+bob_transform_source {
+    name: "validate_install_transform_source",
+    srcs: ["f5.in"],
+    out: {
+        match: "f5.in",
+        replace: ["validate_install_transform_source.txt"]
```

Comment:
Should we put a `$1` in the `replace` value? IMO it's a bit unusual to have a constant value here - if there were multiple sources, then the fixed output file would keep getting overwritten.